src/components/Button/Button.js

```diff
@@ -3,7 +3,7 @@ import { Link } from 'react-router-dom';
 
 import { ReactComponent as PlusIcon } from 'assets/plus-icon.svg';
 
-const Button = React.forwardRef(({ type, icon, children, link, onClick, disabled }, ref) => {
+const Button = React.forwardRef(({ type, icon, children, link, onClick, disabled, shouldOpenNewTab }, ref) => {
```

Comment:
I would say if we are using an `<a>` we should make them all `target='_blank'` no need to create props

Reply:
SGTM and makes sense. All external links would open a new tab. 

It's been removed!